### PR TITLE
*: add store heartbeat message.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @qiuyesuifeng @disksing

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -143,6 +144,20 @@ message AskSplitResponse {
...
 
+message StoreHeartbeatRequest {
+    optional uint64 store_id     = 1;
+    // Capacity for the store.
+    optional uint64 capacity     = 2; 
+    // Available size for the store. 
+    optional uint64 available    = 3;
+    // Total region count in this store. 
+    optional uint32 region_count = 4;
+}
```
Comment (disksing) [inline proto/pdpb.proto]: Do we need to add a `leader_count` in store's heartbeat? PD server need it to do re-balance and it seems not easy for pd-server to count it only by Region's heartbeats.

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM.

**Pull request merged**